src/test/java/teammates/test/cases/webapi/EditInstructorInCourseActionTest.java

```diff
@@ -104,6 +107,31 @@ protected void testExecute() {
         String expectedErrorMessage = new FieldValidator().getInvalidityInfoForEmail(invalidEmail);
         assertEquals(expectedErrorMessage, msg.getMessage());
 
+        ______TS("Failure case: after editing instructor, no instructors are displayed");
+
+        submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, courseId,
+                Const.ParamsNames.INSTRUCTOR_ID, instructorId,
+                Const.ParamsNames.INSTRUCTOR_NAME, instructorToEdit.name,
+                Const.ParamsNames.INSTRUCTOR_EMAIL, invalidEmail,
+
+                Const.ParamsNames.INSTRUCTOR_ROLE_NAME,
+                Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER,
+
+                Const.ParamsNames.INSTRUCTOR_DISPLAY_NAME,
+                Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER,
+
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE, "true",
+                Const.ParamsNames.INSTRUCTOR_IS_DISPLAYED_TO_STUDENT, "false",
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR, "true",
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION, "true",
+                Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT, "true",
+        };
+
+        a = getAction(submissionParams);
+        r = getJsonResult(a);
+        assertEquals(HttpStatus.SC_BAD_REQUEST, r.getStatusCode());
```

Comment:
Not a sufficient check. A `400` code can come from many sources. Check the status message as well.

Reply:
Will update this